### ISSUE TITLE
PopoverContent shows box-shadow in 2.x version 

### ISSUE THREAD

**Issue by abdullahtariq1171** [chakra-ui/chakra-ui#6215]
### Description

Since in 2.x focus outline has been removed but when I use Box component inside `PopoverTrigger` component it still shows box-shadow outline on `PopoverContent`. 

### Link to Reproduction

https://codesandbox.io/s/cool-frost-823v6b?file=/pages/index.tsx

### Steps to reproduce

1. Go to https://codesandbox.io/s/cool-frost-823v6b?file=/pages/index.tsx
2. Click on 'click me' text
3. You'll see the Popover has outline box-shadow 
4. Now comment out the `Box` component inside `PopoverTrigger` and umcomment `Button`, You will notice that it doesn't add outline box-shadow

### Chakra UI Version

2.2.1

### Browser

all

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Hey @abdullahtariq1171,

This is currently showing due to the programmatic focus call within the code. This doesn't work with focus visible for now, consider removing the box shadow with style props.
